vscode/package.json

```diff
@@ -32,10 +32,6 @@
     "node": ">=22.9.0",
     "npm": "^9.5.0 || ^10.5.0"
   },
-  "activationEvents": [
```

Comment:
I'm leaving a comment on this so we can talk about this before this gets merged.

Reply:
This would be better in another small PR so it is easier to track.

Reply:
Moved to #323